[0m02:42:29  Running with dbt=1.9.8
[0m02:42:30  Registered adapter: snowflake=1.9.1
[0m02:42:31  Found 22 models, 108 data tests, 4 seeds, 2 operations, 8 sources, 768 macros
[0m02:42:31  
[0m02:42:31  Concurrency: 4 threads (target='embucket')
[0m02:42:31  
WARNING:snowflake.connector.vendored.urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<snowflake.connector.vendored.urllib3.connection.HTTPConnection object at 0x7f3c1c1f8ce0>: Failed to establish a new connection: [Errno -2] Name or service not known')': /
WARNING:snowflake.connector.vendored.urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<snowflake.connector.vendored.urllib3.connection.HTTPConnection object at 0x7f3c1c1f9730>: Failed to establish a new connection: [Errno -2] Name or service not known')': /computeMetadata/v1/instance/service-accounts/default/email
WARNING:snowflake.connector.vendored.urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<snowflake.connector.vendored.urllib3.connection.HTTPConnection object at 0x7f3c1c2ca1b0>: Failed to establish a new connection: [Errno -2] Name or service not known')': /
WARNING:snowflake.connector.vendored.urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<snowflake.connector.vendored.urllib3.connection.HTTPConnection object at 0x7f3c1c2cbb90>: Failed to establish a new connection: [Errno -2] Name or service not known')': /computeMetadata/v1/instance/service-accounts/default/email
WARNING:snowflake.connector.vendored.urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<snowflake.connector.vendored.urllib3.connection.HTTPConnection object at 0x7f3c14ab91f0>: Failed to establish a new connection: [Errno -2] Name or service not known')': /computeMetadata/v1/instance/service-accounts/default/email
WARNING:snowflake.connector.vendored.urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<snowflake.connector.vendored.urllib3.connection.HTTPConnection object at 0x7f3c14ab8e00>: Failed to establish a new connection: [Errno -2] Name or service not known')': /
[0m02:42:32  1 of 1 START hook: snowplow_web.on-run-start.0 ................................. [RUN]
[0m02:42:32  1 of 1 OK hook: snowplow_web.on-run-start.0 .................................... [[32mOK[0m in 0.02s]
[0m02:42:32  
[0m02:42:32  1 of 22 START sql incremental model public_snowplow_manifest.snowplow_web_base_quarantined_sessions  [RUN]
[0m02:42:32  2 of 22 START sql incremental model public_snowplow_manifest.snowplow_web_incremental_manifest  [RUN]
[0m02:42:32  2 of 22 OK created sql incremental model public_snowplow_manifest.snowplow_web_incremental_manifest  [[32mok 1[0m in 0.50s]
[0m02:42:32  1 of 22 OK created sql incremental model public_snowplow_manifest.snowplow_web_base_quarantined_sessions  [[32mok 1[0m in 0.51s]
[0m02:42:32  3 of 22 START sql table model public_scratch.snowplow_web_base_new_event_limits  [RUN]
WARNING:snowflake.connector.vendored.urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<snowflake.connector.vendored.urllib3.connection.HTTPConnection object at 0x7f3c148806b0>: Failed to establish a new connection: [Errno -2] Name or service not known')': /computeMetadata/v1/instance/service-accounts/default/email
WARNING:snowflake.connector.vendored.urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<snowflake.connector.vendored.urllib3.connection.HTTPConnection object at 0x7f3c1487bf20>: Failed to establish a new connection: [Errno -2] Name or service not known')': /
[0m02:42:32  02:42:32 + Snowplow: New Snowplow incremental model. Backfilling
[0m02:42:33  02:42:33 + Snowplow: Processing data between '2020-01-01 00:00:00' and '2020-01-31 00:00:00' (snowplow_web)
[0m02:42:33  3 of 22 OK created sql table model public_scratch.snowplow_web_base_new_event_limits  [[32mok 1[0m in 0.77s]
[0m02:42:33  4 of 22 START sql incremental model public_snowplow_manifest.snowplow_web_base_sessions_lifecycle_manifest  [RUN]
[0m02:42:34  4 of 22 OK created sql incremental model public_snowplow_manifest.snowplow_web_base_sessions_lifecycle_manifest  [[32mok 1[0m in 0.70s]
[0m02:42:34  5 of 22 START sql table model public_scratch.snowplow_web_base_sessions_this_run  [RUN]
[0m02:42:34  5 of 22 OK created sql table model public_scratch.snowplow_web_base_sessions_this_run  [[32mok 1[0m in 0.50s]
[0m02:42:34  6 of 22 START sql table model public_scratch.snowplow_web_base_events_this_run . [RUN]
[0m02:42:34  6 of 22 OK created sql table model public_scratch.snowplow_web_base_events_this_run  [[32mok 1[0m in 0.40s]
[0m02:42:34  7 of 22 START sql table model public_scratch.snowplow_web_pv_engaged_time ...... [RUN]
[0m02:42:34  8 of 22 START sql table model public_scratch.snowplow_web_pv_scroll_depth ...... [RUN]
[0m02:42:34  9 of 22 START sql table model public_scratch.snowplow_web_sessions_this_run .... [RUN]
[0m02:42:34  10 of 22 START sql incremental model public_derived.snowplow_web_user_mapping .. [RUN]
[0m02:42:35  7 of 22 OK created sql table model public_scratch.snowplow_web_pv_engaged_time . [[32mok 1[0m in 0.40s]
[0m02:42:35  11 of 22 START sql table model public_scratch.snowplow_web_vital_events_this_run  [RUN]
[0m02:42:35  9 of 22 OK created sql table model public_scratch.snowplow_web_sessions_this_run  [[32mok 1[0m in 0.39s]
[0m02:42:35  8 of 22 OK created sql table model public_scratch.snowplow_web_pv_scroll_depth . [[32mok 1[0m in 0.40s]
[0m02:42:35  12 of 22 START sql table model public_scratch.snowplow_web_page_views_this_run . [RUN]
[0m02:42:35  10 of 22 OK created sql incremental model public_derived.snowplow_web_user_mapping  [[32mok 1[0m in 0.49s]
[0m02:42:35  13 of 22 START sql incremental model public_derived.snowplow_web_sessions ...... [RUN]
[0m02:42:35  11 of 22 OK created sql table model public_scratch.snowplow_web_vital_events_this_run  [[32mok 1[0m in 0.30s]
[0m02:42:35  12 of 22 OK created sql table model public_scratch.snowplow_web_page_views_this_run  [[32mok 1[0m in 0.28s]
[0m02:42:35  14 of 22 START sql table model public_scratch.snowplow_web_vitals_this_run ..... [RUN]
[0m02:42:35  15 of 22 START sql incremental model public_derived.snowplow_web_page_views .... [RUN]
[0m02:42:35  14 of 22 OK created sql table model public_scratch.snowplow_web_vitals_this_run  [[32mok 1[0m in 0.29s]
[0m02:42:35  16 of 22 START sql incremental model public_derived.snowplow_web_vitals ........ [RUN]
[0m02:42:36  13 of 22 OK created sql incremental model public_derived.snowplow_web_sessions . [[32mok 1[0m in 0.61s]
[0m02:42:36  17 of 22 START sql table model public_scratch.snowplow_web_users_sessions_this_run  [RUN]
[0m02:42:36  15 of 22 OK created sql incremental model public_derived.snowplow_web_page_views  [[32mok 1[0m in 0.50s]
[0m02:42:36  17 of 22 OK created sql table model public_scratch.snowplow_web_users_sessions_this_run  [[32mok 1[0m in 0.29s]
[0m02:42:36  18 of 22 START sql table model public_scratch.snowplow_web_users_aggs .......... [RUN]
[0m02:42:36  16 of 22 OK created sql incremental model public_derived.snowplow_web_vitals ... [[32mok 1[0m in 0.49s]
[0m02:42:36  19 of 22 START sql table model public_derived.snowplow_web_vital_measurements .. [RUN]
[0m02:42:36  18 of 22 OK created sql table model public_scratch.snowplow_web_users_aggs ..... [[32mok 1[0m in 0.30s]
[0m02:42:36  20 of 22 START sql table model public_scratch.snowplow_web_users_lasts ......... [RUN]
[0m02:42:36  19 of 22 ERROR creating sql table model public_derived.snowplow_web_vital_measurements  [[31mERROR[0m in 0.30s]
[0m02:42:36  20 of 22 OK created sql table model public_scratch.snowplow_web_users_lasts .... [[32mok 1[0m in 0.30s]
[0m02:42:36  21 of 22 START sql table model public_scratch.snowplow_web_users_this_run ...... [RUN]
[0m02:42:37  21 of 22 OK created sql table model public_scratch.snowplow_web_users_this_run . [[32mok 1[0m in 0.30s]
[0m02:42:37  22 of 22 START sql incremental model public_derived.snowplow_web_users ......... [RUN]
[0m02:42:37  22 of 22 OK created sql incremental model public_derived.snowplow_web_users .... [[32mok 1[0m in 0.50s]
[0m02:42:37  
[0m02:42:37  1 of 1 START hook: snowplow_web.on-run-end.0 ................................... [RUN]
WARNING:snowflake.connector.vendored.urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<snowflake.connector.vendored.urllib3.connection.HTTPConnection object at 0x7f3c146e63f0>: Failed to establish a new connection: [Errno -2] Name or service not known')': /
WARNING:snowflake.connector.vendored.urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<snowflake.connector.vendored.urllib3.connection.HTTPConnection object at 0x7f3c146e5610>: Failed to establish a new connection: [Errno -2] Name or service not known')': /computeMetadata/v1/instance/service-accounts/default/email
[0m02:42:37  1 of 1 OK hook: snowplow_web.on-run-end.0 ...................................... [[32mOK[0m in 0.20s]
[0m02:42:37  
[0m02:42:37  Finished running 8 incremental models, 2 project hooks, 14 table models in 0 hours 0 minutes and 6.75 seconds (6.75s).
[0m02:42:38  
[0m02:42:38  [31mCompleted with 1 error, 0 partial successes, and 0 warnings:[0m
[0m02:42:38  
[0m02:42:38    Database Error in model snowplow_web_vital_measurements (models/optional_modules/core_web_vitals/snowflake/snowplow_web_vital_measurements.sql)
  000200: SQL compilation error: unsupported feature: Conflicting ordering requirements in aggregate functions is not supported
  compiled code at target/run/snowplow_web/models/optional_modules/core_web_vitals/snowflake/snowplow_web_vital_measurements.sql
[0m02:42:38  
[0m02:42:38  Done. PASS=23 WARN=0 ERROR=1 SKIP=0 TOTAL=24
